src/main/java/bio/terra/service/rawls/RawlsClient.java

```diff
@@ -49,8 +49,13 @@ public WorkspaceResponse getWorkspace(UUID workspaceId, AuthenticatedUserRequest
       }
       return workspaceCall.getBody();
     } catch (Exception e) {
-      logger.warn("Error retrieving workspace {} by {}", workspaceId, userEmail);
+      logger.warn("Error retrieving workspace", e);
```

Comment:
We previously didn't log the full exception at any point, which made this harder to debug.

Reply:
Good question, I don't think so: Rawls includes the impacted workspace ID in its exceptions, and we can tie logs together via a request ID or trace ID in the MDC if we want more info on the user.  The email is also coming to us via `AuthenticatedUserRequest` which is deprecated, so I felt okay about cleaning that up.